src/cel3ds/3dsread.cpp

```diff
@@ -7,52 +7,58 @@
 // as published by the Free Software Foundation; either version 2
 // of the License, or (at your option) any later version.
 
-#include <iomanip>
+#include <cstdint>
+#include <fstream>
+#include <iostream>
+#include <string>
 
+#include <Eigen/Core>
+
+#include "celutil/bytes.h"
```

Comment:
Hi @ajtribick! We follow the following style: stl, 3rd party libs, cel*/*.h, local *.h. And we use `<>` with cel*.

Reply:
Ok I can change these to angle brackets (I think the order in this file matches what you've stated).

Question - should the celcompat headers (which essentially do forward compatibility for C++17 stl) be grouped with the cel headers or the stl headers?

Reply:
we put them together with other cel headers